Library/Homebrew/utils/formatter.rb

```diff
@@ -50,14 +50,14 @@ def error(string, label: nil)
   # so we always wrap one word before an option.
   # @see https://github.com/Homebrew/brew/pull/12672
   # @see https://macromates.com/blog/2006/wrapping-text-with-regular-expressions/
-  def format_help_text(s, width: 172)
+  def format_help_text(str, width: 172)
     desc = OPTION_DESC_WIDTH
     indent = width - desc
-    s.gsub(/(?<=\S) *\n(?=\S)/, " ")
-     .gsub(/([`>)\]]:) /, "\\1\n    ")
-     .gsub(/^( +-.+  +(?=\S.{#{desc}}))(.{1,#{desc}})( +|$)(?!-)\n?/, "\\1\\2\n#{" " * indent}")
-     .gsub(/^( {#{indent}}(?=\S.{#{desc}}))(.{1,#{desc}})( +|$)(?!-)\n?/, "\\1\\2\n#{" " * indent}")
-     .gsub(/(.{1,#{width}})( +|$)(?!-)\n?/, "\\1\n")
+    str.gsub(/(?<=\S) *\n(?=\S)/, " ")
```

Comment:
```suggestion
    string.gsub(/(?<=\S) *\n(?=\S)/, " ")
```

and this will break indentation below, sorry!

Reply:
No worries, I triggered it 😅